crates/sui-indexer-alt/src/handlers/wal_coin_balances.rs

```diff
@@ -56,4 +57,11 @@ impl Handler for WalCoinBalances {
             .execute(conn)
             .await?)
     }
+
+    async fn prune(from: u64, to: u64, conn: &mut db::Connection<'_>) -> Result<usize> {
+        let filter = wal_coin_balances::table
+            .filter(wal_coin_balances::cp_sequence_number.between(from as i64, to as i64 - 1));
```

Comment:
observation that every prunable table would need an index on cp then. this shouldn't be too concerning though